ext/crypto/lib.rs

```diff
@@ -1322,6 +1368,39 @@ pub async fn op_crypto_decrypt_key(
           .into(),
       )
     }
+    Algorithm::AesCbc => {
+      let key = &*args.key.data;
+      let length = args
+        .length
+        .ok_or_else(|| type_error("Missing argument length".to_string()))?;
+      let iv = args
+        .iv
+        .ok_or_else(|| type_error("Missing argument iv".to_string()))?;
+
+      // 2.
+      let padded_plaintext = match length {
+        128 => decipher_cbc!(aes::Aes128, key, &iv, data),
+        192 => decipher_cbc!(aes::Aes192, key, &iv, data),
+        256 => decipher_cbc!(aes::Aes256, key, &iv, data),
+        _ => unreachable!(),
+      };
+
+      // 3.
+      // padded_plaintext.len() > 0 so this should never panic.
+      let p = *padded_plaintext.last().unwrap() as usize;
```

Comment:
It'd be helpful to explain here that the plaintext is PKCS#7-padded and that the last byte is the size of the padding (and that it's never zero) because I don't think that's very obvious from reading the code itself, only if you know PKCS#7 is used here and you know _what_ PKCS#7 is.